src/components/Header/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Text } from '../Text';
+
+export type HeaderVariant = 'light' | 'dark';
+
+interface Props {
+  variant?: HeaderVariant;
+}
+
+const HeaderWrapper = styled.div`
+  box-sizing: border-box;
+  height: 81px;
+  padding: 8px 11px;
+  background-color: ${({ variant }: Partial<Props>) =>
+    variant === 'light' ? '#ffffff' : '#232323'};
+
+  @media screen and (min-width: 320px) {
```

Comment:
We need to create an issue for making a theme or something where we can set breakpoints and colours.

Reply:
We can use styled-components theme functionality 👍